lib/loaders/api/loader_without_authentication_factory.js

```diff
@@ -60,6 +62,7 @@ export const apiLoaderWithoutAuthenticationFactory = (api, globalAPIOptions = {}
                 },
                 // Cache miss
                 () => {
+                  logger(globalAPIOptions.requestID, apiName, key)
```

Comment:
@orta This logger should count a fetch regardless of cache hit or miss, because in both cases will it lead to a request.

Reply:
Agree, I've got an idea for improvements too.